dftimewolf/lib/timesketch_utils.py

```diff
@@ -32,14 +34,23 @@ def GetApiClient(state):
     assistant = config.ConfigAssistant()
     assistant.load_config_file()
 
-    # TODO: support user supplied passwords to decrypt token file.
-    config.configure_missing_parameters(config_assistant=assistant)
-    ts_client = assistant.get_client()
+    config.configure_missing_parameters(
+        config_assistant=assistant, token_password=token_password)
+
+    ts_client = assistant.get_client(token_password=token_password)
+
+    if not ts_client:
+      state.AddError(
+          'Unable to get a Timesketch API Client', critical=False)
+      return ts_client
```

Comment:
return `None` would probably be better here.

Reply:
done